_posts/2021-05-04-how-to-update-your-ref-arch.adoc

```diff
@@ -0,0 +1,490 @@
+      versions from the Module Catalog than what is shipped with v1.0 of the Reference Architecture. Once you've upgraded to
+      https://github.com/gruntwork-io/infrastructure-modules-multi-account-acme/releases/tag/v0.0.1-20201125[20201125],
+      you'll be at a known starting point and can automate any state manipulations that are required to update a service.
+
+. Make modifications to the `terragrunt.hcl` file to be compatible with the Service Catalog:
+    * Change the `include` path to `find_in_parent_folders("terragrunt_service_catalog.hcl")`. This ensures that you use the
+      Service Catalog compatible root config you created in the previous step.
+    * Change the `terraform.source` attribute to point to the corresponding Terraform module in the
+      `terraform-aws-service-catalog` repo. When updating the source, make sure to set the ref to target `v0.35.4`.
+
++
+[NOTE]
+.Explanation
+====
+This migration guide targets `v0.35.4` of the Service Catalog. Newer versions may require additional state migrations
+that are not covered by the automated scripts. If you wish to update further, first update to `v0.35.4` and then read
+the migration guides in the release notes of the Service Catalog to bump beyond that version.
+====
+
+    * Convert each service listed in `dependencies` to a `dependency` block. Use <<service_migration_guides,the dedicated
+      service migration guide>> as a reference for what dependency blocks are needed.
+    * Update the inputs to adapt to the Service Catalog Terraform module, using the dedicated service migration guide listed
+      below. You can also use `terragrunt validate-inputs` as a sanity check.
+    * Add new required inputs, using `dependency` references as needed.
+    * Remove or rename unused variables.
+
+. Run `terragrunt state list` to sanity check the state backend configuration. You are watching for the following:
+    * You should NOT get any prompts from Terragrunt to create a new S3 state bucket. If you get the prompt, this means that
+      either you are authenticating to the wrong account, or that the bucket name was misconfigured in the root
+      `terragrunt_service_catalog.hcl` file.
+    * You should see resources listed in the state. If the command returns nothing, that means you are not properly linked
+      to the old state file.  Double check the `key` attribute of the `remote_state` block in the root
+      `terragrunt_service_catalog.hcl` config.
+
+. Once you verify the state backend configuration is valid, you will want to perform the state migration operations:
+
+.. *Make a backup of the state file* using the following command: `terragrunt state pull > backup.tfstate
+   --terragrunt-source ...`. You can use this to rollback the state to before you attempted the migration with the
+   following command: `terragrunt state push "$(pwd)/backup.tfstate"`.
```

Comment:
should this be?
```suggestion
.. *Make a backup of the state file* using the following command: `terragrunt state pull --terragrunt-source ... > backup.tfstate`. You can use this to rollback the state to before you attempted the migration with the
   following command: `terragrunt state push --terragrunt-source ..."$(pwd)/backup.tfstate"`.
```

also, what should `--terragrunt-source` be?

Reply:
This is indeed misplaced. Users don't need this. Thanks for catching! 549c027e..e76eb55f